irispy/new_sji.py

```diff
@@ -286,6 +286,42 @@ def dimensions(self):
     def world_axis_physical_types(self):
         return self.cube_like_world_axis_physical_types
 
+    def apply_exposure_time_correction(self, undo=False, force=False):
+        """
+        Applies or undoes exposure time correction to data and uncertainty and adjusts unit.
+
+        Correction is only applied (undone) if the object's unit doesn't (does)
+        already include inverse time.  This can be overridden so that correction
+        is applied (undone) regardless of unit by setting force=True.
+
+        Parameters
+        ----------
+        undo: `bool`
+            If False, exposure time correction is applied.
+            If True, exposure time correction is removed.
+            Default=False
+
+        force: `bool`
+            If not True, applies (undoes) exposure time correction only if unit
+            doesn't (does) already include inverse time.
+            If True, correction is applied (undone) regardless of unit.  Unit is still
+            adjusted accordingly.
+
+        Returns
+        -------
+        result: `None` or `IRISMapCube`
+            If copy=False, the original IRISMapCube is modified with the exposure
+            time correction applied (undone).
+            If copy=True, a new IRISMapCube is returned with the correction
+            applied (undone).
+
+        """
+        new_seq = []
+        for cube in self.data:
+            new_seq.append(cube.apply_exposure_time_correction(undo=undo, force=force))
```

Comment:
This can be written as a list comprehension which is faster for long lists and the same speed for short lists: 
```python
corrected_data = [cube.apply_exposure_time_correction(undo=undo, force=force) for cube in self.data]
```